bedrock/releasenotes/views.py

```diff
@@ -173,7 +167,8 @@ def releases_index(request, product):
         # number has not been used since Firefox 4, then reintroduced with
         # Firefox ESR 24 (Bug 870540). On this index page, 24.1.x should be
         # fallen under 24.0. This pattern is a tricky part.
-        converter = '%g' if int(major_version) in esr_major_versions else '%s'
+        # the outlier is 33.1 which is in major_releases for some reason.
+        converter = '%g' if int(major_version) >= 24 and release != '33.0' else '%s'
```

Comment:
I spend a few minutes trying to make this more pythonic but then `33.1` is really annoying. I came up with

```python
converter = f'{major_version:.0f}' if int(major_version) >= 24 and release not in  ['33.0', '33.1'] else f'{major_version:.1f}'
major_pattern = r'^' + re.escape(converter)
```

which is not great either. Anyway what you have works fine, i's just worried that `round` can bite us.

and also TIL that you can do `rf'{major_version}'` and produce a raw string with variable interpolation.

Reply:
I remember [33.1 ](https://www.mozilla.org/en-US/firefox/33.1/releasenotes/)was some kind of special release that was tied to a marketing campaign (we did a fancy /whatsnew page promoting the "Forget button"). Still it's odd to put it in major releases, but I'm guessing this is probably the reason.

Reply:
Yeah. The call to `round` was confusing for me too. I may play with this some more. It does feel convoluted but I'm guessing there were good reasons for it.

Reply:
I've updated it. I didn't use the `re.escape` call. What problem was that solving? It seems to be working okay without it.

Reply:
Added `re.escape` back in. Of course we need that, the versions have `.` in them 🤦 Good catch.

Reply:
I also threw in a fix for `14.0.1` being duplicated.

Reply:
lgtm!  :)